core/primitives-core/src/profile.rs

```diff
@@ -105,6 +105,27 @@ impl ProfileDataV3 {
     pub fn action_gas(&self) -> Gas {
         self.actions_profile.as_slice().iter().copied().fold(0, Gas::saturating_add)
     }
+
+    /// Returns total compute usage of host calls.
+    pub fn total_compute_usage(&self, ext_costs_config: &ExtCostsConfig) -> Compute {
+        // We currently only support compute costs for host calls. In the future we might add
+        // them for actions as well.
+        self.wasm_ext_profile
+            .iter()
+            .map(|(key, value)| {
+                // Technically, gas cost might be zero while the compute cost is non-zero. To
+                // handle this case, we would need to explicitly count number of calls, not just
+                // the total gas usage.
+                // We don't have such costs at the moment, so this case is not implemented.
+                debug_assert!(key.gas(ext_costs_config) > 0 || key.compute(ext_costs_config) == 0);
```

Comment:
Not an immediate action item, but I would like to hear your opinion here.

Should we aim to (eventually) refactor this to keep counters only? We can always multiply with parameter values when we need the compute or gas costs from the profile, while keeping a burnt and used gas counter that doesn't require as they are today.

Reply:
Totally, I'll look into this as a part of refactoring mentioned below and see if we can reuse/replace `EXT_COSTS_COUNTER` along the way.